D1673/P1673.md

```diff
@@ -6570,48 +6392,48 @@ template<class OtherScalingFactor, class OtherNestedAccessor>
 
   * [3.2]{.pnum} `is_constructible_v<ScalingFactor, OtherScalingFactor>` is `true`.
 
-[3]{.pnum} *Effects:*
+[4]{.pnum} *Effects:*
 
-  * [3.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `other.scaling_factor()`, and
+  * [4.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `other.scaling_factor()`, and
 
-  * [3.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `other.nested_accessor()`.
+  * [4.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `other.nested_accessor()`.
 
 ```c++
 constexpr scaled_accessor(const ScalingFactor& s, const Accessor& a);
 ```
 
-[4]{.pnum} *Effects:*
+[5]{.pnum} *Effects:*
 
-  * [4.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `s`, and
+  * [5.1]{.pnum} Direct non-list-initializes _`scaling-factor`_ with `s`, and
 
-  * [4.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `a`.
+  * [5.2]{.pnum} direct non-list-initializes _`nested-accessor`_ with `a`.
 
 ```c++
 constexpr reference access(data_handle_type p, size_t i) const noexcept;
 ```
 
-[5]{.pnum} *Returns:* `scaling_factor() * Accessor::element_type(` _`nested-accessor`_`.access(p, i))`
+[6]{.pnum} *Returns:* `scaling_factor() * Accessor::element_type(` _`nested-accessor`_`.access(p, i))`
```

Comment:
I think this definition _is_ consistent with the definition of `element_type` above.  `conjugate(d)_scalar` has a consistency issue for its analogous `element_type` and `access`, though.